decidim-core/app/packs/stylesheets/decidim/_redesigned_hero.scss

```diff
@@ -0,0 +1,9 @@
+.hero{
+  @apply h-[370px] md:h-[656px] py-24 flex flex-col justify-center items-center gap-12 md:gap-16;
+
+  background-image: linear-gradient(180deg, rgba(0, 0, 0, 0.7) 21.88%, rgba(0, 0, 0, 0.46421) 60.94%, rgba(0, 0, 0, 0.105) 100%);
```

Comment:
should this be something read from config so its reusable?

Reply:
Actually, it's not being used nowhere else, it's not the same as conferences nor processes/assemblies. IMO, it could follow perfectly the same rule everywhere (the purpose is about enhancing the headings readability, avoiding contrast issues...), but "being the hero", which is something particular itself, perhaps we wanted to keep it unique (in some way).

I dunno, thoughts?